### PR TITLE
RHMAP-11600 - testing jenkins build job

### PR DESCRIPTION
**Pull request by rachael-oregan**
This is a test PR to test the new jenkins job for this service

Some changes were made in sonarqube properties and a dummy test and test folder were created for sonarqube

### CODE REVIEW

**Comment by grdryn** [comment]
Retest this please.

**Comment by grdryn** [comment]
Retest this please.

**Comment by grdryn** [comment]
Retest this please.

**Comment by grdryn** [comment]
Retest this please.

**Comment by grdryn** [comment]
Retest this please.

**Comment by fh-qube** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/fa2f4d2d7be3e01efa14012dbb5a420ddce39c0e/src/app.js#L127) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/fa2f4d2d7be3e01efa14012dbb5a420ddce39c0e/src/app.js#L142) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/fa2f4d2d7be3e01efa14012dbb5a420ddce39c0e/src/app.js#L31) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

**Comment by fh-qube** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/6f9ba55fa8f3d60070f87bd4773e5b95f0171386/src/app.js#L127) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/6f9ba55fa8f3d60070f87bd4773e5b95f0171386/src/app.js#L31) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/6f9ba55fa8f3d60070f87bd4773e5b95f0171386/src/app.js#L142) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

**Comment by fh-qube** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/3d5f12d000a3ee605cfba1418208df35600f6a26/src/app.js#L142) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/3d5f12d000a3ee605cfba1418208df35600f6a26/src/app.js#L127) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/3d5f12d000a3ee605cfba1418208df35600f6a26/src/app.js#L31) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

**Comment by fh-qube** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/a6d32ebcfa4f007d079ded53d959c4d62519d6f1/src/app.js#L31) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/a6d32ebcfa4f007d079ded53d959c4d62519d6f1/src/app.js#L142) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/a6d32ebcfa4f007d079ded53d959c4d62519d6f1/src/app.js#L127) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

**Comment by fh-qube** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/11aee9aa5495ac32b6057916c827508fd9bac230/src/app.js#L142) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/11aee9aa5495ac32b6057916c827508fd9bac230/src/app.js#L31) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/11aee9aa5495ac32b6057916c827508fd9bac230/src/app.js#L127) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

**Comment by fh-qube** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/2a8490bcba4cd46c2f16cd181a7093ec7135ac57/src/app.js#L126) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/2a8490bcba4cd46c2f16cd181a7093ec7135ac57/src/app.js#L141) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)
* MAJOR [Remove this logging statement.](https://github.com/feedhenry/fh-dataman/blob/2a8490bcba4cd46c2f16cd181a7093ec7135ac57/src/app.js#L30) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

File: test/dummy_test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function dummy_test(done) {
```
Comment (pb82) [inline test/dummy_test.js]: Can we add `mocha` to the dependencies and use this for the tests? All new components and tests should be written using mocha.

**Review by pb82 — CHANGES_REQUESTED** [review]

File: test/dummy_test.js (hunk 1 above)
Comment (rachael-oregan) [reply test/dummy_test.js]: Ok I'll add `mocha` in now

File: package.json [hunk 2]
```diff
@@ -35,14 +35,15 @@
...
+    "express-bunyan-logger": "1.3.1",
     "fh-cluster": "0.3.0",
+    "fh-config": "1.1.0",
     "fh-logger": "0.5.1",
-    "cors": "2.8.1",
-    "express-bunyan-logger": "1.3.1",
-    "optimist": "0.6.1"
+    "mongoose": "4.5.1",
+    "optimist": "0.6.1",
+    "sinon": "^1.17.6"
```
Comment (wei-lee) [inline package.json]: I think sinon should be devDependencies?

File: test/dummy_test.js (hunk 1 above)
Comment (wei-lee) [reply test/dummy_test.js]: it's already added

**Comment by rachael-oregan** [comment]
Retest this please

**Comment by rachael-oregan** [comment]
Retest this please

**Review by wei-lee — APPROVED** [review]
LGTM

**Pull request merged**